### PR TITLE
Updates from the Dane FAQ

### PR DESCRIPTION
**Pull request by bradleysigma**
https://docs.google.com/document/d/1_93kw7KkzvAXoQAxRioxo7iAt-Ar6BMIxdNf4Lv25rI

### CODE REVIEW

File: R_wormholes.php [hunk 1]
```diff
@@ -20,6 +20,12 @@
     <li>A system with a wormhole is not an anomaly.</li>
     <li>If a system contains multiple wormholes of the same type, those are treated as separate wormholes.</li>
     <li>A system cannot be adjacent to itself, even if it has multiple wormholes in it.</li>
+    <li>A player chooses the path of adjacent systems each ship moves through during movement. If two systems on that are adjacent because of a wormhole, that ship moves through that wormhole.</li>
+    <ol>
+        <li>If the two systems are adjacent because of a wormhole and because of a shared edge or another reason, the player may choose whether or not the ship moves through the wormhole.</li>
+        <li>If the two systems are adjacent because of two types of wormhole, the player chooses which type of wormhole the ship moves through.</li>
+        <li>A player may move a ship back and forward through a wormhole between two systems any number of times, if its movement value is high enough.</li>
```
Comment (ctinfty) [inline R_wormholes.php:27]:
```suggestion
        <li>A player may move a ship back and forth through a wormhole between two systems any number of times, if its movement value is high enough.</li>
```

**Review by ctinfty — CHANGES_REQUESTED** [review]
Thanks for the promotion 😀 
Please take my suggestions about language with a grain of salt. I'm not a native speaker.

File: C_action_cards.php [hunk 2]
```diff
@@ -389,12 +391,14 @@
...
     <li>A player must have at least one infantry on the game board to play and resolve <i>Refit Troops</i>.</li>
+    <li>If a player has no mechs left in their reinforcements, they may remove a mech from any system that does not contain one of their command tokens and place that instead. The mech will be placed undamaged.</li>
     </ol>
 
 <h1>Reflective Shielding</h1>
     <ol class="note">
     <li>If <i>Reflective Shielding</i> is played after a ship uses its <sc>Sustain Damage</sc> ability during an <b>Assign Hits</b> step, then the hits produced by <i>Reflective Shielding</i> are added to any other hits produced in the immediately proceeding <b>Roll Dice</b> step.</li>
     <ol><li>A unit that used its <sc>Sustain Damage</sc> ability to cancel some of those other hits, and been repaired, cannot use is <sc>Sustain Damage</sc> ability again to cancel the hits produced by <i>Reflective Shielding</i>.</li></ol>
+    <li>If <i>Reflective Shielding</i> is played after a ship uses its <sc>Sustain Damage</sc> ability during any other time (for example, when the Creuss player uses their <i>Dimensional Splicer</i> faction technology ability), the hits are assigned immediately.</li>
```
[the remark this answers is not in the document]
Comment (bradleysigma) [reply C_action_cards.php]: Done.

File: C_technology.php [hunk 3]
```diff
@@ -128,16 +130,17 @@
...
+    <li>Neither ability of <i>Predictive Intelligence</i> may be used if it is exhausted.</li>
     <li>If a player removes any tokens from their fleet pool, they will need to meet their new fleet pool limits before any other effects may resolve.</li>
     </ol>
 
 <h1>Transit Diodes</h1>
     <ol class="note">
     <li>When resolving the ability of <i>Transit Diodes</i>, that player may remove ground forces from a system containing their command token.</li>
-    <li>The ground forces may be on a planet or in a space area.</li>
-    <li>The ground forces may be from different systems.</li>
+    <li>The ground forces may be removed from on a planet or from in a space area.</li>
```
[the remark this answers is not in the document]
Comment (bradleysigma) [reply C_technology.php:140]: Both should work, but the extra words add some emphasis.

File: F_creuss.php [hunk 4]
```diff
@@ -71,7 +72,12 @@
 
 <h1>Sai Seravus <sub>(Commander)</sub></h1>
     <ol class="note">
-    <li>The Creuss player cannot remove a ground force to create capacity for a fighter.</li>
+    <li>The fighters are placed one at a time, until either one has been placed for each ship with capacty that moved through a wormhole, or the capacity of the system is reached.</li>
+    <ol>
+	    <li>The Creuss player cannot remove a ground force to create capacity for a fighter.</li>
```
[the remark this answers is not in the document]
Comment (bradleysigma) [reply F_creuss.php]: Done.

File: F_mahact.php [hunk 5]
```diff
@@ -55,7 +64,7 @@
     <li>Any abilities that trigger when a player moves their ships do not trigger.</li>
     <li>The Mahact player may move another player&rsquo;s ships into an asteroid field only if that player owns the <i>Antimass Deflectors</i> technology.</li>
     <li>The Mahact player may move another player&rsquo;s ships into a supernova only if that player owns the <i>Magmus Reactor</i> Muaat factional technology.</li>
-    <li>The Mahact player may only move the Empyrean player&rsquo;s ships into a nebula.</li>
+    <!--li>The Mahact player may only move the Empyrean player&rsquo;s ships into a nebula.</li-->
```
[the remark this answers is not in the document]
Comment (bradleysigma) [reply F_mahact.php:67]: Until the change is released in a LRR, I'll leave it like this for now.

File: F_mahact.php [hunk 6]
```diff
@@ -68,7 +77,10 @@
 
 <h1>Starlancer <sub>(Mech)</sub></h1>
     <ol class="note">
-    <li>The active player may still use any &ldquo;when/after you activate a system&rdquo; and &ldquo;at the end of your turn&rdquo; abilities.</li>
+    <li>The active player cannot use any &ldquo;when/after you activate a system&rdquo; abilities.</li>
+    <ol><li>Other players cannot use any &ldquo;when/after another player activates a system that&hellip;&rdquo; abilities.</li></ol>
+    <li>The active player may still use any &ldquo;at the end of your turn&rdquo; abilities.</li>
+    <li>The Mahact player may end their own turn and gain their own spent command token from their fleet pool.</li>
```
[the remark this answers is not in the document]
Comment (bradleysigma) [reply F_mahact.php]: Changed to
`<li>The Mahact player may use the ability to end their own turn and gain their own spent command token from their fleet pool.</li>`

File: F_muaat.php [hunk 7]
```diff
@@ -45,7 +46,7 @@
 <h1>Adjudicator Ba&rsquo;al &mdash; Nova Seed <sub>(Hero)</sub></h1>
     <ol class="note">
     <li>Any method of movement is sufficient to trigger this ability, not only tactical action movement (e.g. a retreat).</li>
-    <li>Any command tokens and the Muaat player&rsquo;s units on the replaced system tile are transferred to the new supernova tile.</li>
+    <li>Any command tokens and the Muaat player&rsquo;s units on the replaced system tile are transferred to the new supernova tile. Any Creuss wormhole tokens are returned to the Creuss player. All other tokens are purged.</li>
```
[the remark this answers is not in the document]
Comment (bradleysigma) [reply F_muaat.php]: Done.

File: F_norr.php [hunk 8]
```diff
@@ -40,11 +41,20 @@
...
     <ol><li>This prevents the L1Z1X player from using their <sc>Harrow</sc> faction ability, unless they have the N&rsquo;orr <i>Alliance</i> promissory note and they are the active player.</li></ol>
     <li>Another player may still produce hits against the N&rsquo;orr player&rsquo;s ground forces committed using G&rsquo;hom Sek&rsquo;kus during the <b>Space Cannon Defense</b> step.</li>
-    <li>Effects that prevent or affect movement, such as the N&rsquo;orr player&rsquo;s <i>Ceasefire</i> promissory note or anomalies, do not prevent ground forces from being committed.</li>
+    <li>The N&rsquo;orr player must legally be able to move into the destination system in order to commit ground forces with G&rsquo;hom Sek&rsquo;kus</li>
+    <ol>
+        <li>If another player plays the N&rsquo;orr player&rsquo;s <i>Ceasefire</i> promissory note, they cannot commit any ground forces to planets in the active system during this action.</li>
```
[the remark this answers is not in the document]
Comment (bradleysigma) [reply F_norr.php]: They can. Added this in.

File: F_ul.php [hunk 9]
```diff
@@ -8,12 +8,15 @@
 <h1>Awaken <sub></sub></h1>
     <ol class="note">
     <li>Only sleeper tokens present in the system when it is activated may be replaced with PDS.</li>
+    <li>The Ul player cannot place a PDS on a planet that already has two of their PDS on it. As such, they are unable to place a Hecatoncheries on a planet with two of their PDS using its <sc>Deploy</sc> ability.</li>
```
[the remark this answers is not in the document]
Comment (bradleysigma) [reply F_ul.php]: Done everywhere.

File: F_ul.php [hunk 10]
```diff
@@ -64,6 +75,8 @@
     <ol class="note">
     <li>When the Ul player would place multiple PDS during a single step, they may resolve a <sc>Deploy</sc> ability to place one Hecatoncheries for any or all of those PDS.</li>
     <li>A <sc>Deploy</sc> ability may only be resolved for units in a player&rsquo;s reinforcements. If all four of the Hecatoncheries are already on the board, no Hecatoncheries may be deployed.</li>
+    <li>The Ul player cannot place a PDS on a planet that already has two of their PDS on it. As such, they are unable to place a Hecatoncheries on a planet with two of their PDS using its <sc>Deploy</sc> ability.</li>
```
[the remark this answers is not in the document]
Comment (bradleysigma) [reply F_ul.php:78]: Done.

File: R_nebula.php [hunk 11]
```diff
@@ -16,6 +16,8 @@
     <ol class="note">
     <li>A player cannot retreat (or similar) into a nebula, since it would not be the active system.</li>
     <li>The +1 modifiers for the defender does not modify anti&ndash;fighter barrage, bombardment or space cannon rolls.</li>
+    <li>A future rule change will place the ship movement restrictions of anomalies upon all units.</li>
+    <li>A future rule change will allow ships to move into nebulae when it is not the active system.</li>
```
[the remark this answers is not in the document]
Comment (bradleysigma) [reply R_nebula.php]: Did the former.

File: F_norr.php [hunk 12]
```diff
@@ -47,14 +47,14 @@
...
+        <li>The N&rsquo;orr player cannot commit ground forces to Mirage if it is in a supernova. Another player may do so if they own the <i>Magmus Reactor</i> Muaat factional technology and the N&rsquo;orr player&rsquo;s Alliance promissory note.</li>
+        <li>If the N&rsquo;orr player commits any ground forces from a gravity rift system, those ground forces must roll for removal.</li>
         <li>The N&rsquo;orr player cannot commit ground forces through a wormhole if the <i>Enforced Travel Ban</i> law is in play.</li>
     </ol>
     <li>The N&rsquo;orr player may use this ability to commit ground forces to any planet in the active system, regardless of who, if anyone, controls it.</li>
-    <li>The <i>Dominus Orb</i> relic does not allow the N&rsquo;orr player from committing ground forces from systems containing their command tokens.</li>
+    <li>The <i>Dominus Orb</i> relic does not allow the N&rsquo;orr player to committ ground forces from systems containing their command tokens.</li>
```
Comment (ctinfty) [inline F_norr.php:57]:
```suggestion
    <li>The <i>Dominus Orb</i> relic does not allow the N&rsquo;orr player to commit ground forces from systems containing their command tokens.</li>
```
my bad

File: F_ul.php [hunk 13]
```diff
@@ -71,12 +71,12 @@
     <li>The <sc>Space Cannon</sc> ability is controlled by whichever player controls Elysium.</li>
     </ol>
 
-<h1>Hecatoncheries <sub>(Mech)</sub></h1>
+<h1>Hecatoncheires <sub>(Mech)</sub></h1>
     <ol class="note">
     <li>When the Ul player would place multiple PDS during a single step, they may resolve a <sc>Deploy</sc> ability to place one Hecatoncheries for any or all of those PDS.</li>
```
Comment (ctinfty) [inline F_ul.php:76]:
```suggestion
    <li>When the Ul player would place multiple PDS during a single step, they may resolve a <sc>Deploy</sc> ability to place one Hecatoncheires for any or all of those PDS.</li>
```

File: F_ul.php [hunk 14]
```diff
@@ -71,12 +71,12 @@
...
     <li>A <sc>Deploy</sc> ability may only be resolved for units in a player&rsquo;s reinforcements. If all four of the Hecatoncheries are already on the board, no Hecatoncheries may be deployed.</li>
```
Comment (ctinfty) [inline F_ul.php:77]:
```suggestion
    <li>A <sc>Deploy</sc> ability may only be resolved for units in a player&rsquo;s reinforcements. If all four of the Hecatoncheires are already on the board, no Hecatoncheires may be deployed.</li>
```

**Comment by ctinfty** [comment]
Other than these it looks great!

**Comment by ctinfty** [comment]
I took the liberty to fix those last few myself. Feel free to merge if you're satisfied.

File: F_ul.php (hunk 13 above)
Comment (bradleysigma) [reply F_ul.php]: I could have sworn I did replace all...

**Pull request merged**